src/app/psbt.ts

```diff
@@ -331,6 +338,167 @@ const Psbt = function (config) {
             return psbtModule.broadcastPsbt(psbt);
         },
 
+        signMultipleUtxosPsbtForSend: async ({
+            address, pubKey, selectedUtxos,
+            ownedUtxos, destinationBtcAddress, sendFeeRate
+        }) => {
+            const utxosWithInscription = selectedUtxos.filter(utxo => utxo.inscriptionId);
+            const utxosWithoutInscription = selectedUtxos.filter(utxo => !utxo.inscriptionId);
+            if (utxosWithInscription.length === 0 && utxosWithoutInscription.length === 0) {
+                throw new Error('At least one ordinal or utxo is required.');
+            }
+            const provider = SessionStorage.get(SessionsStorageKeys.DOMAIN);
+            if (provider !== 'alby') {
+                throw new Error('Signing not supported.');
+            }
+
+            const spendableUtxos = ownedUtxos.filter(utxo => !selectedUtxos.some(ownedUtxo => ownedUtxo.txid === utxo.txid))
+                .filter((x) => x.status.confirmed)
+                .filter((x) => x.value > 10000)
+                .filter(utxo => !utxo.inscriptionId)
+                .sort((a, b) => b.value - a.value);
```

Comment:
nit: maybe actually better to do smallest to largest

Reply:
ignore this, nevermind. You are trying to use the largest spendable UTXO as a cardinal if you need to add one

Reply:
Yeah, it uses the biggest first.